src/settings.js

```diff
@@ -435,13 +435,15 @@ const defaults = {
      *
      * @property {number} kSigma - Noise threshold coefficient.
      * @property {boolean} frame - flag, that turns on box frame painting
+     * @property {boolean} IsoMode - flag, that turns on IsoSurface mode instead of Volume Rendering
      *
      */
     VD: {
       kSigma: 1.0,
       kSigmaMed: 2.0,
       kSigmaMax: 4.0,
       frame: true,
+      IsoMode: false,
```

Comment:
Should we start it from a lowercase letter as the implicit code style suggests?

Reply:
Of course, just a misprint